### PR TITLE
New major release: 3185.0.0

### PR DESCRIPTION
**Pull request by dongsupark**
Update SDK version as well as submodules to 3185.0.0.

### CODE REVIEW

**Review by t-lo — APPROVED** [review]
LGTM, thank you Dongsu!

File: sdk_container/.repo/manifests/version.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FLATCAR_VERSION=3187.0.0+nightly-20220323-0139
-FLATCAR_VERSION_ID=3187.0.0
-FLATCAR_BUILD_ID="nightly-20220323-0139"
-FLATCAR_SDK_VERSION=3187.0.0+nightly-20220323-0139
+FLATCAR_VERSION=3185.0.0
+FLATCAR_VERSION_ID=3185.0.0
+FLATCAR_BUILD_ID=""
+FLATCAR_SDK_VERSION=3185.0.0
```
Comment (pothos) [inline sdk_container/.repo/manifests/version.txt:4]: No, this should stay on the nightly

**Review by pothos — CHANGES_REQUESTED** [review]
The main branch does not work with the Alpha SDK in general nor should it use the Alpha submodules

File: sdk_container/.repo/manifests/version.txt (hunk 1 above)
Comment (pothos) [reply sdk_container/.repo/manifests/version.txt:4]: Meaning, we can close this PR

**Pull request closed without merging**